alfredruhara/property-pro-lite
# #167337773  Post an advert property

#### What does this PR do?
It processes the ad data in the database with authorization.
#### Description of Task to be completed? 

- fetch user agent agents informations 

#### How should this be manually tested?

- clone the repository  `https://github.com/alfredruhara/property-pro-lite/` and cd into prop property-pro-lite 

- switch to the branch :ft-create-property-advert-db-postgres-167337773
- run npm start
- make sure you have signed in
```
Provide the token given after signing in
```
then 

- post with this endpoint :localhost:3000/api/v1/property/

#### Any background context you want to provide?
`N/A`
#### What are the relevant pivotal tracker stories?
#167337773
![Screen Shot 2019-07-17 at 9 57 59 AM](https://user-images.githubusercontent.com/50265025/61359108-04b6a300-a87c-11e9-8a0d-c491da695b44.png)
